cms/templates/darklang/preview_lang.html

```diff
@@ -0,0 +1,43 @@
+## Override the default styles_version to the Pattern Library version (version 2)
+<%! main_css = "style-main-v1" %>
+<%page expression_filter="h"/>
+<%! from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<%inherit file="../base.html" />
+
+<%block name="title">Preview Language Setting</%block>
+
+<%block name="bodyclass">is-signedin course uploads view-container</%block>
+
+<%block name="content">
+<h1>
+    ${Text(_("Preview Language Setting"))}
+</h1>
+
+<div class="">
+    <form class="form" action="/update_lang/" method="post">
+        <fieldset class="form-group">
+            <div class="field">
+                <label class="field-label" for="preview_lang">${Text(_("Language Code"))}</label>
+                <input class="field-input input-text" type="text" id="preview_lang" name="preview_lang"
+                       placeholder="e.g. en" aria-describedby="Input abbreviation for your language of choice" />
+            </div>
+        </fieldset>
+        <div class="form-actions">
+            <button class="btn-brand btn-base" type="submit" name="set_language" value="set_language">${Text(_("Submit"))}</button>
+            <button class="btn-brand btn-base" type="submit" name="reset" value="reset">${Text(_("Reset"))}</button>
+        </div>
+
+        <input type="hidden" name="csrfmiddlewaretoken" value="${ csrf_token }"/>
+    </form>
+
+    <br/>
+    % if not form_submit_message is UNDEFINED:
```

Comment:
It doesn't work to translate a variable-- in order for the text to be scraped for translation, the actual text needs to be present within _(..).